src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_PaymentOptionsScreen.tsx

```diff
@@ -89,11 +90,11 @@ export const Profile_PaymentOptionsScreen = ({
       addPaymentMethodCallbackHandler,
     );
     return () => eventSubscription.remove();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [appStatus, authorizeRecurringPayment, cancelRecurringPayment]);
```

Comment:
Do we need `appStatus` here?

Reply:
again, I followed the vipps login example, and we can remove it here, tested on dev build.

Reply:
refactored